ui/tests/integration/components/primary-metric/allocation-test.js

```diff
@@ -63,6 +63,22 @@ module('Integration | Component | PrimaryMetric::Allocation', function(hooks) {
     assert.equal(findAll('[data-test-chart-area]').length, mockTasks.length);
   });
 
+  test('memory chart shows max if enabled', async function(assert) {
+    await preload(this.store);
+
+    this.server.create('job', {
+      groupCount: 1,
+      resourceSpec: ['M: 256, MM: 512'],
+    });
+    const alloc = this.server.create('allocation');
+
+    this.setProperties({ alloc, metric: 'memory' });
+    await render(template);
+
+    console.log(find('.annotation').innerText);
+    assert.ok(find('.annotation').innerText.includes('/ 256 MiB reserved, 512 MiB max'));
```

Comment:
I've struggled with getting this test to pass - I always get `0 MiB / 0 MiB reserved`.  I get the same text in the above test as well.  Would love pointers for how to set it up.

Reply:
Apparently, the issue was a bit gnarlier than that - the tracker spy/mock didn't actually fill-in the annotation info, and there is something odd about the mirage creation if multiple jobs/annotations are in play.  Anyways, I took out the test, and would love to have a frontend engineer take over if possible :( sorry!